src/Module.purs

```diff
@@ -101,3 +103,22 @@ prependFolder :: Folder -> File -> File
 prependFolder (Folder folder) (File file) = File (folder <> "/" <> file)
 
 infixr 5 prependFolder as </>
+
+findM
+   :: forall m f a b t
+    . Alt t
+   => Monad m
+   => Foldable f
+   => f a
+   -> (a -> m (t b))
+   -> t b
+   -> m (t b)
+findM xs f base =
+   foldr step (pure base) xs
+   where
+   step :: a -> m (t b) -> m (t b)
```

Comment:
Idiomatically `step` can be written
```purescript
   step :: a -> m (t b) -> m (t b)
   step x acc = acc `lift2 (<|>)` f x
```
I’ve made this change directly.

Reply:
Also I’ve moved this to `Util` as it’s not particularly related to `Module` (and might use elsewhere).